src/WebJobs.Script.WebHost/FileMonitoringService.cs

```diff
@@ -137,6 +137,8 @@ private void InitializeNonBlockingFileWatchers()
             {
                 if (!_watchersStopped)
                 {
+                    FileUtility.EnsureDirectoryExists(_hostLogPath);
```

Comment:
Just making sure this does not check a directory outside of wwwroot during cold start, can you confirm please?

Reply:
Yes, this code is executed in the background after a delay, as part of Ankit's previous coldstart work.